src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -1058,6 +1060,10 @@ private unsafe void DisposeInternal()
             exceptionAggregator.ThrowIfNeeded();
         }
 
+        protected virtual void DisposePeriodicDocumentsMigrator(ExceptionAggregator exceptionAggregator)
```

Comment:
we might add other stuff here along the way, so let's rename it to...
`DisposeBackgroundWorkers`

Reply:
and put `TombstoneCleaner` & friends there